clients/drcachesim/common/options.cpp

```diff
@@ -640,3 +640,11 @@ droption_t<bool> op_enable_drstatecmp(
     DROPTION_SCOPE_CLIENT, "enable_drstatecmp", false, "Enable the drstatecmp library.",
     "When true, this option enables the drstatecmp library that performs state "
     "comparisons to detect instrumentation-induced bugs due to state clobbering.");
+
+#ifdef BUILD_PT_TRACER
+droption_t<bool> op_enable_kernel_tracing(
+    DROPTION_SCOPE_ALL, "enable_kernel_tracing", false, "Enable Kernel PT tracing.",
+    "By default, offline tracing only records userspace trace. If this option is "
+    "enabled, offline tracing will record syscall's Kernel PT and write every syscall's "
```

Comment:
"PT" is an Intel-specific feature and not everyone will know what  it is.  Explain that it only works on Intel processors that support the feature.

Reply:
Done.